src/utilities/animations.scss

```diff
@@ -181,3 +181,17 @@
     transform: scale(1.025);
   }
 }
+
+/* Rotate an element 360 degrees over and over, used for spinners */
+.anim-rotate-360 {
+  animation: rotate-360-keyframes 1s linear infinite;
+}
+
+@keyframes rotate-360-keyframes {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(359deg);
```

Comment:
Is there a benefit of using `359deg` instead of `360deg`? Doesn't it add a very small "jump" at the end?

Reply:
:+1: I've used the existing, simpler `anim-rotate` since it's shorter and looks the same (and as you point out, is probably more accurate too).